### PR TITLE
Code Improvements

### PR DESCRIPTION
**Pull request by dmitrolov**

### CODE REVIEW

File: src/sass/variables.sass [hunk 1]
```diff
@@ -3,8 +3,9 @@ $base-color-text-grey: #9b9b9b
...
 $arrow-color: #E6E6E6
 
-$sm: 320px
-$md: 768px
-$lg: 1024px
-$xl: 1440px
-$xxl: 2560px
\ No newline at end of file
+// Screen resolutions
+$screen-resolution-mobile: 320px
```
Comment (LenaAviles) [inline src/sass/variables.sass]: $mobile, $tablet-portrait, $tablet-landscape, $desktop, $large-desktop

File: src/sass/mixins.sass [hunk 2]
```diff
@@ -1,3 +1,11 @@
+@import "variables"
+
 @mixin arwTransform($prop)
   transform: rotate($prop)
   -webkit-transform: rotate($prop)
+
+@mixin upper-navigation-tablet-up
```
Comment (LenaAviles) [inline src/sass/mixins.sass]: I really don't like this mixin, nor name, nor position, nor functionality

File: src/sass/header.sass [hunk 3]
```diff
@@ -141,7 +135,7 @@
                     &:hover
                         color: white
             .menu.lvl3
```
Comment (LenaAviles) [inline src/sass/header.sass]: maybe it's better to create new menu, because this classes is not supported BEM

File: src/sass/variables.sass (hunk 1 above)
Comment (dmitrolov) [reply src/sass/variables.sass]: Changed to
`@mixin for-tablet-up
  @media (min-width: $tablet-portrait)
    @content`

`@include for-tablet-up
        flex-direction: row
        text-align: center
        padding: 1em`

File: src/sass/header.sass [hunk 4]
```diff
@@ -7,7 +7,10 @@
 .upper-navigation
     display: flex
     flex-direction: column
-    @include upper-navigation-tablet-up
+    @include for-tablet-up
```
Comment (dmitrolov) [inline src/sass/header.sass]: > I really don't like this mixin, nor name, nor position, nor functionality

File: src/sass/mixins.sass (hunk 2 above)
Comment (dmitrolov) [reply src/sass/mixins.sass]: [And what about this code?](https://github.com/JuliaKH/arsenal/pull/31/commits/5b524a59b75cff78d5ff70e23e5a1b4e63a9d2c8?file-filters%5B%5D=.css&file-filters%5B%5D=.map&file-filters%5B%5D=.sass#r300267334)
[](url)

File: src/sass/variables.sass (hunk 1 above)
Comment (dmitrolov) [reply src/sass/variables.sass]: [Done](https://github.com/JuliaKH/arsenal/pull/31/commits/5b524a59b75cff78d5ff70e23e5a1b4e63a9d2c8?file-filters%5B%5D=.css&file-filters%5B%5D=.map&file-filters%5B%5D=.sass#r300268855)

File: src/sass/header.sass (hunk 3 above)
Comment (dmitrolov) [reply src/sass/header.sass]: Should I make another menu without foundation?

**Pull request merged**